// skilstak-colors/index.js
'use strict';

var choice = require('skilstak-choice').choice;

module.exports.base03 = '\x1b[1;30m';
module.exports.b03 = '\x1b[1;30m';
module.exports.base02 = '\x1b[0;30m';
module.exports.b02 = '\x1b[0;30m';
module.exports.base01 = '\x1b[1;32m';
module.exports.b01 = '\x1b[1;32m';
module.exports.base00 = '\x1b[1;33m';
module.exports.b00 = '\x1b[1;33m';
module.exports.base0 = '\x1b[1;34m';
module.exports.b0 = '\x1b[1;34m';
module.exports.base1 = '\x1b[1;36m';
module.exports.b1 = '\x1b[1;36m';
module.exports.base2 = '\x1b[0;37m';
module.exports.b2 = '\x1b[0;37m';
module.exports.base3 = '\x1b[1;37m';
module.exports.b3 = '\x1b[1;37m';
module.exports.yellow = '\x1b[0;33m';
module.exports.y = '\x1b[0;33m';
module.exports.orange = '\x1b[1;31m';
module.exports.o = '\x1b[1;31m';
module.exports.red = '\x1b[0;31m';
module.exports.r = '\x1b[0;31m';
module.exports.magenta = '\x1b[0;35m';
module.exports.m = '\x1b[0;35m';
module.exports.violet = '\x1b[1;35m';
module.exports.v = '\x1b[1;35m';
module.exports.blue = '\x1b[0;34m';
module.exports.b = '\x1b[0;34m';
module.exports.cyan = '\x1b[0;36m';
module.exports.c = '\x1b[0;36m';
module.exports.green = '\x1b[0;32m';
module.exports.g = '\x1b[0;32m';

module.exports.clear = '\x1b[H\x1b[2J';
module.exports.cl = '\x1b[H\x1b[2J';

module.exports.reset = '\x1b[0m';
module.exports.x = '\x1b[0m';

module.exports.colors = [
  module.exports.yellow,
  module.exports.orange,
  module.exports.red,
  module.exports.magenta,
  module.exports.violet,
  module.exports.blue,
  module.exports.cyan,
  module.exports.green,
];

module.exports.random = function () {return choice(module.exports.colors)};
module.exports.rc = module.exports.random;
